lib/identity/stateTransitions/identityCreateTransition/validateIdentityCreateSTStructureFactory.js

```diff
@@ -22,7 +25,14 @@ function validateIdentityCreateSTStructureFactory(
       rawIdentityStateTransition = identityStateTransition;
     }
 
-    const result = new ValidationResult();
+    const result = validator.validate(
```

Comment:
the structure validates in https://github.com/dashevo/js-dpp/blob/602a386d55e2ecd90836f0d132dd2c9b01c04475/lib/stateTransition/validation/validateStateTransitionStructureFactory.js#L58

Reply:
That doesn't seem to work then since I can just pass a random object and it returns no errors

Reply:
Wait, the function you send the link to is actually calling one you've commented